AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+||ziffstatic.com/jst/zdconsent.js
```

Comment:
I see another url
https://cdn.ziffstatic.com/jst/eu/zdconsent.js
I think it's better to add this domain to:
`###_evidon-barrier-wrapper`

Reply:
Okay.

Reply:
Fixed.